cloudbuild.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 Google Inc.
+#
+# Redistribution and use in source and binary forms, with or without
```

Comment:
Please use Apache 2.0 license (you can copy from vcf_to_bq.py)

Reply:
Done!